### PR TITLE
added more tests

### PR DESCRIPTION
**Pull request by zjthomas**

### CODE REVIEW

**Comment by zjthomas** [comment]
@lbgraham can you give this a quick look over when you've got a chance

File: client/components/Nav.jsx [hunk 1]
```diff
@@ -1,29 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const left = (
-  <svg id="48x48_chevron_left" height="48" viewBox="0 0 48 48" width="48">
-    <path d="M29.414 5.992c.566 0 1.137.192 1.614.588 1.115.925 1.296 2.613.404 3.77L20.902 24l10.53 13.65c.892 1.156.71 2.844-.404 3.77-1.116.924-2.743.737-3.635-.42L15.57 25.675a2.76 2.76 0 0 1 0-3.35L27.394 6.998a2.548 2.548 0 0 1 2.02-1.008z" />
-  </svg>
-);
+const Nav = ({ direction, clickHandler }) => {
+  const left = (
```
Comment (lbgraham) [inline client/components/Nav.jsx]: Cool. Didn't know you could do it like this.

File: client/components/__test__/App.test.js [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
+    global.fetch = jest.fn(() => Promise.resolve(sampleData));
+  });
+
+  it('renders without crashing', () => {
     const wrapper = shallow(<App />);
     expect(toJson(wrapper)).toMatchSnapshot();
   });
+
+  it('has a function called scoll that increments and decrements the currentPhoto property in state', () => {
+    const wrapper = shallow(<App />);
```
Comment (lbgraham) [inline client/components/__test__/App.test.js]: Is it typical to include all of these expects in one 'it' block or should they be broken up?

File: client/testSetup.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
-configure({ adapter: new Adapter() });
+configure({ adapter: new Adapter(), disableLifecycleMethods: true });
```
Comment (lbgraham) [inline client/testSetup.js]: Yep.

File: database/seed.js [hunk 4]
```diff
@@ -1,95 +1,6 @@
...
-    photo_id: 'aZPpi_jgFCxO--HGyBpbTQ',
-    user_id: '_geqzETHmwyh4TGSHJcOrQ',
-    user_photo_id: 'NInuR_ZyzR5hlOBchRidww',
-    caption: '',
-    label: 'food',
-  },
-
-];
+const sampleData = require('./sampleData');
+const Photo = require('./photos');
```
Comment (lbgraham) [inline database/seed.js]: Did you mean to keep this file if you've moved everything out of it?

**Pull request merged**